app/src/components/Card/Card.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { render, cleanup } from "@testing-library/react";
+import Card from "./Card";
+
+describe("Card", () => {
+  afterEach(() => cleanup);
+
+  describe("className", () => {
+    const customClass = "custom-card-class";
+
+    it(`should render with custom class`, async () => {
```

Comment:
why not `"should render with custom class"` here and everywhere :)

Reply:
true, good catch :D